db/migrate/20190926144242_add_service_inventories_table.rb

```diff
@@ -0,0 +1,30 @@
+class AddServiceInventoriesTable < ActiveRecord::Migration[5.2]
+  def change
+    create_table :service_inventories, :id => :bigserial, :force => :cascade do |t|
+      t.references :tenant, :index => true, :null => false, :foreign_key => {:on_delete => :cascade}
+      t.references :source, :index => false, :null => false, :foreign_key => {:on_delete => :cascade}
+
+      t.string :source_ref, :null => false
+      t.string :name
+      t.string :inventory_type
```

Comment:
What are some examples of inventory_type?  Wondering if this should be in extra

Reply:
hm, maybe? In the account I have it's all `inventory`, so not sure what the types are

Reply:
I say this would distinct e.g. AWS inventory, inventory as a script, etc.

Reply:
I'm not sure what that means :)

Reply:
ah ok, so it's basically binary now, let me move it to extra

`Type: Identifies whether it is a standard inventory or a Smart Inventory.`

https://docs.ansible.com/ansible-tower/latest/html/userguide/inventories.html#smart-inventories

Reply:
:+1:

Reply:
done